sensor.py

```diff
@@ -6,10 +6,13 @@
 from custom_components.smartthinq import (
     CONF_LANGUAGE, KEY_SMARTTHINQ_DEVICES, LGDevice)
 import homeassistant.helpers.config_validation as cv
-from homeassistant.const import CONF_REGION, CONF_TOKEN
+from homeassistant.const import CONF_REGION, CONF_TOKEN, DEVICE_CLASS_POWER, POWER_WATT
 
-import wideq
-from wideq import dishwasher
+from . import wideq
+from .wideq import dishwasher
+from .wideq import ac
+
+REQUIREMENTS = ['https://github.com/dacrypt/wideq/archive/master.zip#wideq==1.3.1']
```

Comment:
I could be wrong, but I thought this was already enforced in the top-level module?